src/components/basicComponents/BaseButton.vue

```diff
@@ -46,7 +55,7 @@ defineProps({
     type: String,
     required: true,
     validation: (variant) =>
-      ["primary", "secondary", "neutral", "focused"].includes(variant),
+      ["primary", "secondary", "neutral", "voted"].includes(variant),
```

Comment:
Don't kill me but voted doesn't sound right too. Is it sth like small variant or transparent?

Reply:
no it is just his function - this is the button where you vote for the feedback you want that would be implemented. You're right it's not reusable. I must think about better name.

Reply:
this is a small button so I changed the name for small